source/extensions/omni.isaac.lab/omni/isaac/lab/utils/timer.py

```diff
@@ -118,6 +127,9 @@ def stop(self):
         self._elapsed_time = self._stop_time - self._start_time
         self._start_time = None
 
+        if self._name:
+            Timer.timing_info[self._name] = self._elapsed_time
```

Comment:
Just thinking: Do we also want to store the count -- number of times that timer was called? That makes it simple to compute the average?

Reply:
hmm I thought it just captures a one-time elapsed time?

Reply:
ah nvm, you mean like storing a list of the times with the same name?